packages/sit-onyx/src/components/OnyxKey/OnyxKey.stories.ts

```diff
@@ -0,0 +1,96 @@
+import type { Meta, StoryObj } from "@storybook/vue3-vite";
+import { CANONICAL_KEYS } from "../../utils/shortcut.js";
+import OnyxKey from "./OnyxKey.vue";
+
+/**
+ * The key component displays keyboard keys with proper OS-specific symbols and accessibility support.
+ * Useful for documentation, tutorials, and keyboard shortcut displays.
+ */
+const meta: Meta<typeof OnyxKey> = {
+  title: "Support/Key",
+  component: OnyxKey as Meta["component"],
+  tags: ["unstable"],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof OnyxKey>;
+
+/**
+ * This example shows a default key.
+ */
+export const Default = {
+  args: {
+    keyName: "enter",
+  },
+} satisfies Story;
+
+/**
+ * Pressed state visualization.
+ */
+export const Pressed = {
+  args: {
+    keyName: "enter",
+    pressed: true,
+  },
+} satisfies Story;
+
+/**
+ * This example shows a skeleton key.
+ */
+export const Skeleton = {
+  args: {
+    skeleton: true,
+    keyName: "enter",
+  },
+} satisfies Story;
+
+/**
+ * Example: simple shortcut.
+ */
+export const Shortcut = {
+  render: (args) => ({
+    components: { OnyxKey },
+    setup: () => ({ args }),
+    template: `
+      <div style="display: flex; gap: 0.5rem; align-items: center; color: var(--onyx-color-text-icons-neutral-medium);">
+        <OnyxKey v-bind="args" keyName="ctrl" />
+        <span>+</span>
+        <OnyxKey v-bind="args" keyName="shift" />
+        <span>+</span>
+        <OnyxKey v-bind="args" keyName="C" />
+      </div>
+    `,
+  }),
+  args: {
+    variant: "auto",
+  },
+} satisfies Story;
+
+/**
+ * Displays all supported canonical keys from CANONICAL_KEYS.
+ * Useful for visual verification of the mapping.
+ */
+export const AllCanonicalKeys = {
+  render: (args) => {
+    const keys = CANONICAL_KEYS.filter((key) => key !== "unknown");
+
+    return {
+      components: { OnyxKey },
+      setup: () => ({ args, keys }),
+      template: `
+        <div style="display: flex; flex-wrap: wrap; gap: 0.5rem;">
+          <OnyxKey
+            v-for="key in keys"
+            :key="key"
+            v-bind="args"
+            :keyName="key"
+          />
+        </div>
+      `,
+    };
+  },
+  args: {
+    variant: "macOS",
+  },
+} satisfies Story;
```

Comment:
```suggestion
/**
 * Displays all supported canonical keys from CANONICAL_KEYS.
 * Useful for visual verification of the mapping.
 */
export const AllCanonicalKeys = createAdvancedStoryExample(
  "OnyxKey",
  "CanonicalKeysExample",
) satisfies Story;
```

The downside of using render functions in Storybook is that the generated code snippet in the docs does not reflect the actual code written here. To solve this, we created a `createAdvancedStorybookExample()` utility function. With it, you can create a dedicated .vue file inside e.g. OnyxKey/examples/CanonicalKeysExample.vue and it will then be rendered as story as well as including the correct source code from the vue file as code snippet